modules/rosa-mobb-prereq-checklist.adoc

```diff
@@ -0,0 +1,121 @@
+
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-sts-aws-prereqs.html
+
+
+:_content-type: PROCEDURE
+[id="rosa-mobb-prereq-checklist_{context}"]
+= Prerequisites checklist to deploy a ROSA classic cluster 
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-15
+//---
+//date: '2023-07-27'
+//title: Prerequisites Checklist to Deploy ROSA Cluster with STS 
+//tags: ["ROSA", "STS"]
+//authors:
+//  - Byron Miller
+//  - Connor Wooley
+//  - Diana Sari
+//---
+
+This is a checklist of prerequisites needed to spin up a classic [product-title] cluster with links:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[STS]. 
```

Comment:
this didn't render correctly on the preview url

![image](https://github.com/openshift/openshift-docs/assets/2488346/d009b197-3903-49ad-94aa-21cc4b997f10)